README.md

```diff
@@ -437,6 +438,17 @@ repository.records.first.sets
 
 Return an `Array` of `String` [set specs](#fieldhandsetspec) indicating set memberships of this record.
 
+#### `Fieldhand::Record#dump`
```

Comment:
Hard problem: what _should_ we call this method?

I believe Ox goes with `dump` for some symmetry with [`Marshal`](https://ruby-doc.org/core-2.2.2/Marshal.html#method-c-dump) (as it allows you to "dump" any Ruby object into XML) but perhaps our use case is closer to `to_xml` as used by Oga and Nokogiri?